courses/free-courses-hi.md

```diff
@@ -410,6 +411,15 @@
 * [Bootstrap Tutorial in Hindi With 2 Projects for Beginners \| Complete Bootstrap 5 Tutorial in Hindi](https://youtube.com/watch?v=QE5oQh63gGE) - Tech Gun
 
 
+#### Material UI
+
+* [Learn Reactjs material ui - Hindi](https://www.youtube.com/watch?v=0euw01oikEw&ab_channel=TechinfoYT) - Techinfo YT
+* [Material UI Complete in One Video (Hindi)](https://www.youtube.com/watch?v=TJz6y9RLtA8&ab_channel=GeekyShows) - Geeky Shows
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes